.github/workflows/test_bettertransformer.yml

```diff
@@ -0,0 +1,37 @@
+name: BetterTransformer / Python - Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        os: [ubuntu-20.04, macos-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install .[tests]
+        pip3 install --upgrade torch torchvision torchaudio
+        pip install --upgrade git+https://github.com/huggingface/accelerate.git@main
+        pip install --upgrade git+https://github.com/huggingface/transformers.git@main
```

Comment:
Nope, it just means that we're up-to-date with the latest version of `transformers` so that we can flag anything if someone changed something on the modeling script of `transformers`

Reply:
I would advocate using transformers stable version. I think it's the standard practice no? evaluate, accelerate, optimum seem to do that

Reply:
Yes, fixed in 570c2cc033420dd69922baa142e3e364f315fa41